packages/robusta_firebase_cloud_messaging/example/test/main_test.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:robusta_firebase_cloud_messaging_example/main.dart';
+
+void main() {
+  group('test app', () {
+    testWidgets('can run app', (tester) async {
+      await expectLater(runner.run, returnsNormally);
```

Comment:
It should be `await expectLater(runner.run(), completes);`

Reply:
Done.